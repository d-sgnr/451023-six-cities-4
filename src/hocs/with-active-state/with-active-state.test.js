import React from "react";
import renderer from "react-test-renderer";
import withActiveState from "./with-active-state.jsx";

const MockComponent = () => <div/>;

const MockComponentWrapped = withActiveState(MockComponent);

it(`withActiveState is rendered correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped
      isActive={false}
      onActiveChange={() => {}}
    />
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
