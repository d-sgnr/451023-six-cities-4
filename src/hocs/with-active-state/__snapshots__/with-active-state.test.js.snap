// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withActiveState is rendered correctly 1`] = `<div />`;
